src/index.html

```diff
@@ -4,10 +4,218 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500,700;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="header__action"><a class="header__link" href="#">Call to order</a></div>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">Home</li>
+          <li class="nav__item"><a class="nav__link" href="#">About us</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Gallery</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Consultation</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Testimonials</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="section">
+      <div class="info">
+        <h1 class="info__heading">
+          Miami <strong>best</strong><br> Real estate <a class="link" href="#">here</a>
+        </h1>
+        <div class="info__text">
+          <p><strong>Over 50 years experience in love.</strong><br>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p>The market leader in real estate since 2003.<br>Awards Real Estate Company, 2010, 2011, 2013.<br>More than 200 satisfied customers premium segment.<br>We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.</p>
+          <p><strong>We will never be late!<br>Guaranteed secure transaction - 100%.</strong></p>
+        </div>
+      </div>
+    </section>
+    <section class="section">
+      <h2 class="heading section__heading">
+        Gallery<span class="heading__crown"></span>
+      </h2>
+      <div class="gallery">
+        <div class="photos photos__wide_left">
+          <img
+            class="photos__img"
+            src="./images/photos/1.jpg"
+            alt="1"
```

Comment:
```suggestion
            alt="house"
```
alt має відповідати змісту фотографії